File: blockchain/blockchain/rnd.js
var Web3 = require('web3');


// https://rpc.apothem.network

// xdcB0B2C7B14ba9A552E819b9799Ee54bD2E0E94717

console.log("Hello world!")

var MyContract = [
    {
      "inputs": [],
      "name": "retrieve",
      "outputs": [
        {
          "internalType": "uint256",
          "name": "",
          "type": "uint256"
        }
      ],
      "stateMutability": "view",
      "type": "function"
    },
    {
      "inputs": [
        {
          "internalType": "uint256",
          "name": "num",
          "type": "uint256"
        }
      ],
      "name": "store",
      "outputs": [],
      "stateMutability": "nonpayable",
      "type": "function"
    }
  ]

const init = async () => {
    const web3 = new Web3("https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161");
    const contract = new web3.eth.Contract(MyContract,"0x70242ade66d64ac7f2245238b2c7622462e7c221");
    const id = await new web3.eth.net.getId();
    console.log(id);
    contract.methods.store(10).call()
    var result = await contract.methods.retrieve().call();
    console.log(result);
    console.log("Connected");
}

init();

File: blockchain/blockchain/rnd.py
import time
from unittest import result
from brownie import Contract, network, web3


abi = {
    "0x8b8c9c14c8e1442e90dd6ff82bb9889ccfe5a54d88ef30776f11047ecce5fedb": [
      {
        "inputs": [],
        "name": "retrieve",
        "outputs": [
          {
            "internalType": "uint256",
            "name": "",
            "type": "uint256"
          }
        ],
        "stateMutability": "view",
        "type": "function"
      },
      {
        "inputs": [
          {
            "internalType": "uint256",
            "name": "num",
            "type": "uint256"
          }
        ],
        "name": "store",
        "outputs": [],
        "stateMutability": "nonpayable",
        "type": "function"
      }
    ]
  }          # contract ABI as a dict
address = "xdc224beCd8F4238c248cD69aC68f0EE4AF053a06F6"  # contract address as a string


network.connect('https://rpc.apothem.network')
my_contract = Contract("Storage", address, abi=abi)

height = 38989777

# while True:
my_contract.store(10) # change to call the desired method
result = my_contract.retrieve()
print(result)
    # do whatever you need to do
# while web3.eth.blockNumber == height:
#     time.sleep(1)
#     height = web3.eth.blockNumber